widgetssdk/src/main/java/com/glia/widgets/helper/rx/RxUtil.kt

```diff
@@ -0,0 +1,60 @@
+package com.glia.widgets.helper.rx
+
+import io.reactivex.CompletableTransformer
+import io.reactivex.ObservableTransformer
+import io.reactivex.SingleTransformer
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.disposables.Disposable
+import io.reactivex.schedulers.Schedulers
+
+/**
+ * Helpful rx tools to simplify subscriptions and disposals
+ *
+ * For the schedulers, the use is through the compose() function in the subscription flow.
+ *
+ * Example
+ * observable.compose(observableSchedulers()).subscribe(...)
+ */
+
+fun <T> observableSchedulers(): ObservableTransformer<T, T> {
```

Comment:
I think from the function's name it is unclear which thread rx should subscribe and observe.
It would be better to have something like `Observable.ioMainThreadScheduler()`, `Single.ioMainThreadScheduler()` etc.